test/fixtures/models/katello_activation_keys.yml

```diff
@@ -45,3 +45,14 @@ library_dev_staging_view_key:
   environment_id: <%= ActiveRecord::FixtureSet.identify(:dev) %>
   content_view_id: <%= ActiveRecord::FixtureSet.identify(:library_dev_staging_view) %>
   auto_attach: true
+
+key_purpose_attributes:
```

Comment:
we don't need to name this 'key_' ... we already know it's an activation key :)

Reply:
Yes, you are correct. But I'm following a pattern which exists. I renamed it to purpose_attributes_key.